app/controllers/films_controller.rb

```diff
@@ -46,6 +46,12 @@ def destroy
     end
   end
 
+  def delete_film_link_attachment
```

Comment:
What about move it to separate controller? Something like `AttachmentsController` in Film `namespace` (`Film::AttachmentsController`), where you can make a `delete` method - in my opinion it will be more RESTful and routes will look like this:
```ruby
namespace :film do
  resources :attachment, only: :delete
end
```

Reply:
it is similar to the previous Service. At the moment is only one method so maybe not worth to move it yet?

Reply:
It always starts from 1 simple method, try to keep your controller clean from the beginning